rust/lon/src/cli.rs

```diff
@@ -65,6 +65,10 @@ enum Commands {
     Bot {
         #[clap(subcommand)]
         commands: BotCommands,
+        #[arg(long, default_value_t = false)]
+        detailed: bool,
+        #[arg(short, default_value_t = 50)]
+        num_commits: usize,
```

Comment:
I wouldn't want to parameterize via the CLI here. Let's move these directly into bot_fallible.

The bot should always be configured via env variables as it's convenient to set them in the yaml files and the rest of the CLI should primarily be configured via cmdline flags.